packages/caliper-fabric-ccp/lib/fabricNetwork.js

```diff
@@ -154,6 +155,8 @@ class FabricNetwork {
 
         this.mutualTls = CaliperUtils.checkProperty(this.network, 'mutual-tls') ? this.network['mutual-tls'] : false;
 
+        this.useGateway = CaliperUtils.checkProperty(this.network, 'use-gateway') ? this.network['use-gateway'] : false;
```

Comment:
Instead of the network config entry, something like this should be used in the `fabric.js` constructor:
https://github.com/hyperledger/caliper/blob/27ec5f94e70a4cb3856f2b1dfd26b3e2f95b087f/packages/caliper-fabric-ccp/lib/fabric.js#L166